lib/dal/src/component.rs

```diff
@@ -3332,6 +3332,10 @@ impl Component {
         ctx.add_dependent_values_and_enqueue(component.input_socket_attribute_values(ctx).await?)
             .await?;
 
+        Geometry::restore_all_for_component_id(ctx, component_id)
```

Comment:
The reason you went here vs. expanding the `import_component_subgraph` is because the geometries aren't in the subgraph of the component?

Reply:
@britmyerss yeah, the subgraph code is very generic. Since we've structured our geometries the other way around I had to implement a special case here